cmd/server/main.go

```diff
@@ -1,20 +1,37 @@
 package main
 
 import (
+	filesystemapi "github.com/kubernetes-csi/csi-proxy/internal/os/filesystem"
 	"github.com/kubernetes-csi/csi-proxy/internal/server"
-	filesystem "github.com/kubernetes-csi/csi-proxy/internal/server/filesystem"
+	filesystemsrv "github.com/kubernetes-csi/csi-proxy/internal/server/filesystem"
+	srvtypes "github.com/kubernetes-csi/csi-proxy/internal/server/types"
+	flag "github.com/spf13/pflag"
+)
+
+var (
+	kubeletCSIPluginsPath = flag.String("kubelet-csi-plugins-path", `C:\var\lib\kubelet\plugins`, "Absolute path of the Kubelet plugin directory in the host file system")
+	kubeletPodPath        = flag.String("kubelet-pod-path", `C:\var\lib\kubelet\pods`, "Absolute path of the kubelet pod directory in the host file system")
 )
 
 func main() {
-	s := server.NewServer(apiGroups()...)
+	flag.Parse()
+	apiGroups, err := apiGroups()
+	if err != nil {
+		panic(err)
+	}
+	s := server.NewServer(apiGroups...)
 	if err := s.Start(nil); err != nil {
 		panic(err)
 	}
 }
 
 // apiGroups returns the list of enabled API groups.
-func apiGroups() []server.APIGroup {
-	return []server.APIGroup{
-		&filesystem.Server{},
+func apiGroups() ([]srvtypes.APIGroup, error) {
+	fssrv, err := filesystemsrv.NewServer(*kubeletCSIPluginsPath, *kubeletPodPath, filesystemapi.New())
+	if err != nil {
+		return []srvtypes.APIGroup{}, err
```

Comment:
Fine for now, but going forward we might want to have a `Config` struct, and have a standard init func taking a `Config` and returning a `(srvtypes.APIGroup, error)` tuple, so we can daisy chain inits.

More of a note for later, good for now :)